docs/rate-limits.md

```diff
@@ -71,8 +71,8 @@ per account, per hostname, per hour. This limit will be higher on staging so you
 can use staging to debug connectivity problems.
 
 The "new-reg", "new-authz" and "new-cert" endpoints have an **Overall
-Requests** limit of 20 per second. All other endpoints have an Overall Requests
-limit of 2000 per second.
+Requests** limit of 20 per second. The "/directory" endpoint and the "/acme" 
+directory & subdirectories have a limit of 40 requests per second.
```

Comment:
Let's say "have an Overall Requests limit of 40 requests persecond." Thanks for working on this!